File: II/CSharp/laba10/laba9/laba9/laba3/Accessory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace laba3
{
    public class Accessory : Product, IName<Accessory>
    {
        protected string brand;
        protected string color;

        public Accessory()
        {
            brand = " ";
            color = " ";

        }
        public Accessory(string name, double price, string brand, string color) : base(name, price)
        {
            this.brand = brand;
            this.color = color;
        }

        public string Brand
        {
            get
            {
                return brand;
            }
            set
            {
                brand = value;
            }
        }

        public string Color
        {
            get
            {
                return color;
            }
            set
            {
                color = value;
            }
        }

        public int CompareTo(Accessory other)
        {
            throw new NotImplementedException();
        }

        public override string ToString()
        {
            return $"{base.ToString()},Brand:{brand}, Color:{color}";

        }
    }
}

File: IV/Programming in computer networks/Laba1/src/Accessory.java
public class Accessory extends Product{
private String brand;
private String color;

Accessory(){
    super();
    brand=null;
    color=null;
}
Accessory(String n,double p,String b, String c){
    super(n,p);
    brand=b;
    color=c;
}
void setBrand(String b){
    brand=b;
}
String getBrand(){
    return brand;
}
void setColor(String c){
    color=c;
}
String getColor(){
    return color;
}
public String toString(){
    return super.toString()+", Бренд: "+brand+",Цвет: "+color;
}
}
